src/test/scala/za/co/absa/hyperdrive/trigger/persistance/SchedulerInstanceRepositoryTest.scala

```diff
@@ -92,4 +92,14 @@ class SchedulerInstanceRepositoryTest
     val result = await(schedulerInstanceRepository.getAllInstances())
     result should contain theSameElementsAs TestData.schedulerInstances
   }
+
+  "getCurrentDateTime" should "return database current date time" in {
+    val result = await(schedulerInstanceRepository.getCurrentDateTime())
+    val expected = LocalDateTime.now()
```

Comment:
Hmm, you could get unlucky with that. What if `schedulerInstanceRepository.getCurrentDateTime()` is executed at e.g. 2022-12-31 23:59:59.999

What you could do is take the `LocalDateTime.now()` before invoking `schedulerInstanceRepository.getCurrentDateTime()` and after it and assert that the result is in between.

Reply:
Fixed